gigascience/gigadb-website
# A11y - Search page headings and hero layout

# Pull request for issue: #1372

This is a pull request for the following functionalities:

* Setup correct heading hierarchy (no skipping heading levels)
* Update styles of hero layout to avoid relying on html elements for spacing

## How to test?

Navigate to http://gigadb.gigasciencejournal.com:9170/search/new?keyword=Genomic&type%5B%5D=dataset&dataset_type%5B%5D=Genomic and run AXE extension to verify that headings levels are not skipped

## How have functionalities been implemented?

* Setup more semantic heading for filter partial
* Decouple of semantic html and styles by using css class (.h4) to define text style
* Remove empty `<p>` which was used in hero for spacing, and use css properties to achieve same effect

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
